VAULT-45104: Added beta warning and updated vault version required (#2932)
hashicorp/terraform-provider-vault: website/docs/r/agent_registration.html.md

# vault\_agent\_registration

> ~> **Beta feature:** This feature is currently available as a preview and is possibly incomplete and subject to change. **We strongly discourage using preview or beta features with production workflows.**

Reviewer: ```suggestion
~> **Beta feature:** This feature is currently in beta status and is possibly incomplete and subject to change. **We strongly discourage using preview or beta features with production workflows.**
```
